entgql/annotation.go

```diff
@@ -45,6 +45,8 @@ type (
 		Implements []string `json:"Implements,omitempty"`
 		// Directives to add on the field/type.
 		Directives []Directive `json:"Directives,omitempty"`
+		// QueryField allow you expose the Entity on the query as a field.
```

Comment:
```suggestion
		// QueryField exposes the generated type with the given string under the Query object.
```

Can we also make the name optional, so a call to `QueryField()` will expose the type with generated name (plural form)?